### PR TITLE
Enhancement/my stuff state

### PR DESCRIPTION
**Pull request by Figments**
this ended up being more about alerts lmao, but it's gotten too big so it's time for a review

### CODE REVIEW

File: packages/client/src/app/shared/alerts/components/alerts/alerts.component.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        {{ data.message }}
+    </div>
+    <div *ngSwitchCase="alertType.Error" class="alert">
+        <span class="alert-icon"><i-feather name="x"></i-feather></span>
+        <span class="alert-title">Uh-oh.</span>
+        {{ data.message }}
+    </div>
+    <div *ngSwitchCase="alertType.Info" class="alert">
+        <span class="alert-icon"><i-feather name="info"></i-feather></span>
+        <span class="alert-title">Head's up:</span>
```
Comment (pingzing) [inline packages/client/src/app/shared/alerts/components/alerts/alerts.component.html]: Should be "Heads up:"

File: packages/client/src/app/shared/alerts/components/alerts/alerts.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                document.documentElement.style.setProperty('--alert-color', 'rgb(60,128,40)');
+                break;
+            case AlertType.Error:
+                document.documentElement.style.setProperty('--alert-color', '#DD4C4F');
+                break;
+            case AlertType.Info:
+                document.documentElement.style.setProperty('--alert-color', 'rgb(80,154,233)');
+                break;
+            case AlertType.Warning:
+                document.documentElement.style.setProperty('--alert-color', '#d9a107')
```
Comment (pingzing) [inline packages/client/src/app/shared/alerts/components/alerts/alerts.component.ts]: Can these be consistent in their color formatting? Either `rbg()` or hex values for all of them.

**Review by pingzing — CHANGES_REQUESTED** [review]

File: packages/client/src/app/shared/alerts/components/alerts/alerts.component.ts (hunk 2 above)
Comment (Figments) [reply packages/client/src/app/shared/alerts/components/alerts/alerts.component.ts]: i'll make them all rgb then

File: packages/client/src/app/shared/alerts/components/alerts/alerts.component.html (hunk 1 above)
Comment (Figments) [reply packages/client/src/app/shared/alerts/components/alerts/alerts.component.html]: lmao dumb mistake

**Review by pingzing — APPROVED** [review]

**Pull request merged**